lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -33,12 +34,34 @@ def create_inference_job(
     return job_response
 
 
-@router.post("/evaluate/", status_code=status.HTTP_201_CREATED)
-def create_evaluation_job(
+@router.post("/annotate/", status_code=status.HTTP_201_CREATED)
+def create_annotation_job(
     service: JobServiceDep,
-    job_create_request: JobEvalCreate,
+    job_create_request: JobAnnotateCreate,
     request: Request,
-    response: Response
+    response: Response,
+) -> JobResponse:
+    # Lumigator's opinion on the best summarization model
+    # and the one that should generate annotations.
+    # In the future, we could expose this via a config file
+    # so that, for a supported task, there is a default
+    # "reference" model. For now, we keep the current functionality:
+    # Lumigator decides who annotates.
+
+    inference_job_create_request = JobInferenceCreate(
```

Comment:
this should be `JobAnnotateCreate` if I'm understanding the intent correctly, otherwise if there's not enough of a distinction between Annotate and create to create its own class, we should forego JobIAnnotateCreate and stick with JobInferenceCreate, wdyt?

Reply:
Just in case: the intent is to take the fields from the annotate job (= a subset of an inference job), complete with the "opinionated" fields, so to speak, and launch that opinionated inference job.

At first I considered sticking to JobInferenceCreate. The reason I didn't in the end was it meant exposing a field that we would later overwrite, which is kind of lying to the user (the user asks for mistral in their config file and then gets annotations by bart... that's not nice) and a silent failure risk.

So what we do here is create that opinionated InferenceCreate config with the AnnotateCreate fields plus the ones Lumigator mandates.

Does this make more sense?